docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "2"
+services:
+  practitioner-service:
+    build: ./practitioner-service
+    image: ghcr.io/ndph-arts/practitioner-service:${GITHUB_SHA}
+    depends_on:
+      - fhir-api
+    environment:
+      SERVER_PORT: "80"
+    ports:
+      - "80:80"
+
+  fhir-api:
+    image: "marrobi/r4-fhir-server"
```

Comment:
marrobi ? is that the official one?

Reply:
not my file. will be resolved in other PR